lstmcpipe/onsite_mc_r0_to_dl3.py

```diff
@@ -102,31 +101,15 @@
 
     # 1 STAGE --> R0/1 to DL1
     if 'r0_to_dl1' in stages_to_run:
-
-        if workflow_kind == 'lstchain':
-
-            log_batch_r0_dl1, debug_r0dl1, jobs_all_r0_dl1 = batch_r0_to_dl1(
-                dl0_data_dir,
-                abspath(args.config_file_lst),
-                prod_id,
-                all_particles,
-                source_env=source_env,
-                gamma_offsets=gamma_offs
-            )
-
-        elif workflow_kind == 'hiperta':
-
-            log_batch_r0_dl1, debug_r0dl1, jobs_all_r0_dl1 = batch_r0_to_dl1_rta(
-                dl0_data_dir,
-                abspath(args.config_file_rta),
```

Comment:
To clarify a bit more, I then I see 2 ways of launching the full workflow. 

In case of `ctapipe` and `lstchain`:
`python onsite_mc_r0_dl3.py -c config_MC_prod.yml -conf_lst lstchain_*.json`

and in case of HiPeRTA:
`python onsite_mc_r0_dl3.py -c config_MC_prod.yml -conf_lst lstchain_*.json -conf_rta hiperta_config.yml`